assets/scss/components/_organisation-logo.scss

```diff
@@ -32,6 +44,20 @@ a.organisation-logo {
   }
 }
 
+/*
+Stacked
+
+Text appears below the logo.
+
+Markup:
+<div class="organisation-logo-stacked {{modifier_class}}">
+HMRC Revenue & Customs
```

Comment:
can you also add a `<span>` in the markup example as demonstrated by the styles on line `69` and `111`